cypress.json

```diff
@@ -0,0 +1,11 @@
+{
+    "baseUrl": "http://localhost:8080",
+    "reporter": "spec",
+    "viewportHeight": 600,
```

Comment:
Maybe FHD?

Reply:
What would be the gain ?

Reply:
I see magic numbers I propose new ones :P 
1920*1080 seems closer to real life application to me.